README.md

```diff
@@ -134,3 +251,68 @@ It returns a text file in working directory.
     For different type of software versions the raw data path could be different for the same
     concept. For example, Nanonis software `generic 5e` has `/Piezo Configuration/Active Calib.`
     and generic 4.5 has `/Piezo Calibration/Active Calib.` for the same concept `/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib`.
+  - In the config file the concept that takes data from ELN is denoted by `@eln` otherwise data will come from experimental raw files.
```

Comment:
```suggestion
  - In the config file the concept that takes data from ELN is denoted by the token `@eln`. Otherwise, data will come from raw experimental files.
```

Reply:
Done!